### DISCUSSION TITLE
How can I tell GitHub Desktop that I moved my editor?

### DISCUSSION THREAD

**Discussion by ASRFS** [community/community#22816] [New to GitHub]
<p>I don’t know how I managed it, but when I installed Atom on my Mac I put it in Downloads. Sometime after that, I installed GitHub Desktop, and it happily bound itself to Atom.</p>
<p>Then I realized I didn’t <em>want</em> Atom in Downloads and moved it to Applications where it belongs.</p>
<p>Now GitHub Desktop can’t find it, so when I click “Open in Atom” it gives me an error message with the old location. I can’t find any configuration options that allow me to tell it where Atom lives these days.</p>
<p>Is there some way to fix this? It’s not the end of the world, but it is annoying.</p>
<p>Thanks!</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/asrfs">@ASRFS</a> welcome! Did you try restarting GitHub Desktop? That normally should resolve this issue, and if not there are some other things we can try to get this fixed.</p>

**Comment by ASRFS** [comment]
<p><a class="mention" href="/steveward">@steveward</a> - That did it! Thanks so much.</p>
